src/dune_rules/mdx.ml

```diff
@@ -2,6 +2,15 @@ open Import
 
 let mdx_version_required = "1.6.0"
 
+let color_always =
+  lazy
+    (if Lazy.force Ansi_color.stderr_supports_color then
+     [ Command.Args.A "--color=always" ]
+    else [])
+
+let verbosity =
+  lazy (if !Log.verbose then [ Command.Args.A "--verbosity=info" ] else [])
```

Comment:
Tbh, I find it kind of weird that dune's own verbosity settings would affect mdx. Perhaps there should be an environment variable in mdx to control this?